src/benefits/load.go

```diff
@@ -44,7 +45,11 @@ func (q *ServiceStruct) GetAll(lang string) []models.Benefits {
 // GetByID returns a Benefit from an ID
 func (q *ServiceStruct) GetByID(lang, benefitId string) (models.Benefits, error) {
 	for _, benefit := range q.GetAll(lang) {
-		if benefit.ID == benefitId {
+		val, err := strconv.Atoi(benefitId)
```

Comment:
benefitId can be passed in as an int instead of converting

Reply:
See above